apps/glusterfs/cluster_entry.go

```diff
@@ -218,6 +218,10 @@ func (c *ClusterEntry) DeleteBricksWithEmptyPath(tx *bolt.Tx) error {
 
 	for _, nodeid := range c.Info.Nodes {
 		node, err := NewNodeEntryFromId(tx, nodeid)
+		if err == ErrNotFound {
+			logger.Warning("Ignoring nonexisting node [%v] in "+
+				"cluster [%v].", nodeid, c.Info.Id)
+		}
```

Comment:
Did you mean to add a `continue` statement to this code block?

Reply:
Gosh, thanks! Done...

Reply:
Maybe I should add unit tests for this new behavior?!... :-)

Reply:
I'd be all in favor of that. Or if you wish you could add to the functional tests for upgrade @ tests/functional/TestUpgrade